### ISSUE TITLE
No longer starts up on user login.

### ISSUE THREAD

**Issue by palladin9479** [Rem0o/FanControl.Releases#2646]
None of the fan control v198 releases startup automatically when the user logs in.  I have even put a link to the exe in the startup folder, made it run as admin, everything and it still will not start up.  I revert back to v196 and startup works again.  Not sure what was changed but likely a regression bug.

**Comment by tobbissimo** [comment]
same. Starts automatically when computer starts, but not when awaking from sleep.

**Comment by Rem0o** [comment]
Startup works by creating a scheduled task in the Windows Task Scheduler.  Mind taking a look at the underlying task?

**Comment by maru801** [comment]
> same. Starts automatically when computer starts, but not when awaking from sleep.

Same here. I can confirm that waking my PC from sleep makes fan control close.
I need to reopen it every time I wake from sleep.

My PC is running Windows 10. I'm currently on V198 too, but noticed this has been happening with the last update as well.

**Comment by AnonymousWP** [comment]
Yes I can reproduce this; after sleep, FanControl has exited. When I start it, configuration is loaded again.

**Comment by BledaMundzuk** [comment]
Same here. Tried several work-arounds to no avail

**Comment by Rem0o** [comment]
Any crash dump in EventViewer ?

**Comment by Rem0o** [comment]
Are you guys all on AMD gpus? Which kind?

**Comment by AnonymousWP** [comment]
> Are you guys all on AMD gpus? Which kind?

7800XT.

**Comment by Rem0o** [comment]
Driver version for Adrenaline/ADLX also would be appreciated.

**Comment by AnonymousWP** [comment]
> Driver version for Adrenaline/ADLX also would be appreciated.

24.5.1. Where do I find the version of ADLX?

**Comment by maru801** [comment]
> Are you guys all on AMD gpus? Which kind?

7900XT
Adrenaline v24.5.1

**Comment by Rem0o** [comment]
Are you all staying on 24.5.1 because of bugs on games with the newer drivers?  All my testing I've been doing lately on 24.7.1

**Comment by AnonymousWP** [comment]
> Are you all staying on 24.5.1 because of bugs on games with the newer drivers?  All my testing I've been doing lately on 24.7.1

That's right. 24.6.1 had stability issues according to some users on a forum and Reddit. 24.7.1 has a known issue regarding reduced performance on Battlefield 2042. That's why I'm staying on 24.5.1 for now.

**Comment by maru801** [comment]
> Are you all staying on 24.5.1 because of bugs on games with the newer drivers?  All my testing I've been doing lately on 24.7.1

I've just been lazy and haven't updated yet. I'll update later and see if it fixes the issue.

**Comment by BledaMundzuk** [comment]
> Are you guys all on AMD gpus? Which kind?

Latest Nvidia driver here

**Comment by Asmod4n** [comment]
> Startup works by creating a scheduled task in the Windows Task Scheduler. Mind taking a look at the underlying task?

that doesn't work with windows 11. you mustn't use UAC for it to work. the only solution is to write a windows service and give that system rights.

**Comment by maru801** [comment]
Updating my PC to the latest AMD Adrenaline version seems to have fixed the issue.

**Comment by GLWine** [comment]
The problem also persists on version 199, even inserting the program into the pinification utility does not change.
